apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/StorageActivity.java

```diff
@@ -79,7 +83,6 @@ public void onItemClick(AdapterView<?> parent, View view, int position, long id)
             }
         });
 
-
         /* TODO remove reflection once Storage published to jCenter. */
```

Comment:
You can address TODO now as we split project/jcenter code so no need for reflection

Reply:
Fix in new commit.